src/dvc_objects/fs/implementations/gdrive.py

```diff
@@ -161,20 +162,8 @@ def fs(self):
         from pydrive2.auth import GoogleAuth
         from pydrive2.fs import GDriveFileSystem as _GDriveFileSystem
 
-        temporary_save_path = self._gdrive_user_credentials_path
```

Comment:
`temporary_save_path` was only used in `service_account` mode, so there was no point of setting it here.